docs/examples/sqg.rst

```diff
@@ -34,7 +34,7 @@ the buoyancy, :math:`b`, is hydrostatic balance
 .. math::
 
 
-   b = - \partial_z \psi. 
+   b = \partial_z \psi. 
 
  Using the fact that the Potential Vorticity is exactly zero in the
```

Comment:
The problem is that line 39 is indented. So it is interpreted as still part of the math block. I am surprised you didn't get this problem. Did you run ``make html`` to test your documentation changes?